src/features/accept-bid/AcceptBidButton/AcceptBidButton.tsx

```diff
@@ -1,10 +1,12 @@
 import { FC } from 'react';
 
-import { AcceptBidModal, AcceptBidModalProps } from '..';
+import { Bid } from '@zero-tech/zauction-sdk';
+
+import { AcceptBidModal } from '..';
 
 export interface AcceptBidButtonProps {
-	zna: AcceptBidModalProps['zna'];
-	bid: AcceptBidModalProps['bid'];
+	zna: string;
```

Comment:
The `type` changes within this PR were previously discussed with Brett. If creating a sub component, then the `types` should be from the parent component. These were previously typed from the child component (wrong way round)